packages/core/src/functions.ts

```diff
@@ -30,10 +31,13 @@ export interface EvalValueData {
     tsVarOverride?: Record<string, string> | null;
     cssVarsMapping?: Record<string, string>;
     args?: string[];
+    rootArgument?: string;
+    evaluatorNode?: postcss.Node;
```

Comment:
why is the new `evaluatorNode` used over the already passed down `node` - is `node` overridden at some nested flow?

Reply:
`evaluatorNode` - probably a more fitting name is `initialNode` and it's used to give initial node as context.